src/libraries/trimming/trimming.csproj

```diff
@@ -0,0 +1,83 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>$(NetCoreAppCurrent)-$(TargetOS)</TargetFramework>
+
+    <RuntimeIdentifier>$(PackageRID)</RuntimeIdentifier>
+    <PublishTrimmed>true</PublishTrimmed>
+    <UseAppHost>true</UseAppHost>
+
+    <TrimmerDefaultAction>link</TrimmerDefaultAction>
+    <TrimMode>link</TrimMode>
+    <SuppressTrimAnalysisWarnings>false</SuppressTrimAnalysisWarnings>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <!-- Exclude packaging wrapper projects that don't produce a library -->
+    <ProjectExclusions Include="$(LibrariesProjectRoot)Microsoft.Extensions.HostFactoryResolver\src\*.csproj" />
+    <ProjectExclusions Include="$(LibrariesProjectRoot)System.Composition\src\*.csproj" />
+    <!--Exclude projects that don't have a net6.0 compatible configuration-->
+    <ProjectExclusions Include="$(LibrariesProjectRoot)System.Private.Runtime.InteropServices.JavaScript\src\*.csproj" />
+    <ProjectExclusions Include="$(LibrariesProjectRoot)Microsoft.Diagnostics.Tracing.EventSource.Redist\src\*.csproj" />
+    <ProjectExclusions Include="$(LibrariesProjectRoot)Microsoft.IO.Redist\src\*.csproj" />
+    <!--Exclude Microsoft.NETCore.* assemblies-->
+    <ProjectExclusions Include="$(LibrariesProjectRoot)Microsoft.NETCore.Platforms\src\*.csproj" />
+    <ProjectExclusions Include="$(LibrariesProjectRoot)Microsoft.NETCore.Targets\src\*.csproj" />
+    <!--Exclude single Tests project that doesn't follow default naming conventions of the repo-->
+    <ProjectExclusions Include="$(LibrariesProjectRoot)Microsoft.Extensions.DependencyInjection.Specification.Tests\src\*.csproj" />
+    <!--Exclude Source Generators-->
+    <ProjectExclusions Include="$(LibrariesProjectRoot)Microsoft.XmlSerializer.Generator\src\*.csproj" />
+    <!--Exclude Common Project-->
+    <ProjectExclusions Include="$(LibrariesProjectRoot)Common\src\*.csproj" />
+
+
+    <Compile Include="Program.cs" />
```

Comment:
Is that file actually needed? Could this be a .proj instead that uses the Traversal SDK similar to src.proj?

Reply:
Needs to be a console app in order to be able to be fully trimmed, and if it is a console app, then it needs to have a main, so yes the file is needed.